lib/peg.d.ts

```diff
@@ -466,6 +466,14 @@ export namespace parser {
    */
   function parse(grammar: string, options?: Options): ast.Grammar;
 
+  type OutputType =
+    | "ast"
+    | "parser"
+    | "source-and-map"
+    | "source-with-inline-map"
+    | "source"
+    ;
+
```

Comment:
I think I missed something here... I should probably be using SourceBuildOptions<SourceOutputs> rather than Options. Although SourceOutputs doesn't include `"ast"`. I'll take a look...

Reply:
I pushed a change that reverts this, and uses `SourceBuildOptions<SourceOutputs>` instead